pynamodb/connection/base.pyi

```diff
@@ -58,6 +58,7 @@ class Connection:
     def get_item_collection_map(self, return_item_collection_metrics): ...
     def get_exclusive_start_key_map(self, table_name: Text, exclusive_start_key): ...
 
+    def get_operation_kwargs_for_delete_item(self, *args, **kwargs) -> Dict: ...
```

Comment:
Are those `get_operation_kwargs_for_...` becoming part of the API?

Reply:
do you have a different suggestion? i thought it would be easier to have specific functions so that people don't pass parameters that don't work with that operation

Reply:
actually, now that i've made the function `get_operation_kwargs` I can delete/replace all of these operation-specific ones